src/i18n/fr.js

```diff
@@ -0,0 +1,44 @@
+export default {
+  header: {
+      slogan: 'Le futur du développement web'
+  },
+  callToAction: {
+      start: 'Premier pas',
+      join: 'Discuter'
+  },
+  nav: {
+      code: 'Code',
+      cloud: 'Cloud',
+      docs: 'Docs'
+  },
+  bio: {
+      first: '<a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">Ultra-rapide</a>, Asynchrone, architecture "event-driven" et propulsé par <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO by Apple</a>.',
```

Comment:
Does `Asynchrone` need to be capitalized?

Reply:
Nop